prediction_market_agent/agents/microchain_agent/microchain_agent.py

```diff
@@ -56,10 +57,13 @@ def build_agent(
         temperature=0.7,
     )
     agent = Agent(llm=LLM(generator=generator), engine=engine)
+    agent.max_tries = 3
     agent.prompt = SYSTEM_PROMPT.format(engine_help=engine.help)
     agent.bootstrap = [
         'Reasoning("I need to reason step by step. Start by assessing my '
-        'current position and balance.")'
+        "current positions and balance. Do I have any positions in the markets "
+        "returned from GetMarkets? Consider selling overvalued tokens AND "
+        'buying undervalued tokens.")'
```

Comment:
That's great - nudging in the right direction seems like a great approach

Another thing we could explore is to provide more concrete examples with actual numbers, like on AutoTx (https://github.com/polywrap/AutoTx/blob/main/autotx/agents/SendTokensAgent.py#L23)

Reply:
Good idea. Will leave to later a PR if that's okay